resources/META-INF/plugin.xml

```diff
@@ -42,9 +42,13 @@
     <applicationService serviceImplementation="com.intellij.lang.jsgraphql.ide.references.GraphQLFindUsagesUtil" />
 
     <!-- Language registration -->
-    <fileTypeFactory implementation="com.intellij.lang.jsgraphql.GraphQLFileTypeFactory"/>
+    <fileType name="GraphQL" implementationClass="com.intellij.lang.jsgraphql.GraphQLFileType" fieldName="INSTANCE" language="GraphQL"
```

Comment:
it works only for versions ~ >=2019.2  as far as I know

Reply:
I increased the minimum version to 2019.2. Given that the new 2020.3 version is coming out in the next few weeks, supporting 2018 and 2019.1 seems unreasonable. This will significantly complicate further maintenance and compatibility with 2020 in terms of removing deprecated and using newly created APIs.

https://plugins.jetbrains.com/docs/marketplace/product-versions-in-use-statistics.html

@jimkyndemeyer what do you think?

Reply:
@vepanimas It's difficult to say without knowing how many users would be affected.

The majority of the users are coming from WebStorm, and I get the feeling that they usually run newer versions to get the latest JS goodies. On the flip side, I also see Android Studio users, and it'd be a shame to cut them off from updates.

If I understand the link you posted correctly, 2018 support is needed given that "the latest major release, as well as at least two previous major releases" is where 80% of user installs are compatible.

Do you have internal numbers for this specific plugin to make an informed decision?

Reply:
@jimkyndemeyer there is some confusion in the definitions, I also misunderstood at first. A major release means a branch number from [here](https://jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html#intellij-platform-based-products-of-recent-ide-versions), not a year. So for the most recent 202 (2020.2) the rule "latest - 3" looks like 202, 201, 193 and 192.

Reply:
@vepanimas Ah, sounds good to me then.